pkg/os/os.go

```diff
@@ -0,0 +1,114 @@
+package os
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	Log "github.com/deis/deis/pkg/log"
```

Comment:
Upper-case package name? Why not name the package something like `logger` and just import it here?